infinispan-remote/src/test/resources/storedprocedures/resultSetProcedure.js

```diff
@@ -0,0 +1,7 @@
+// mode=local,language=javascript,parameters=[id,title]
```

Comment:
Hi @ntviet18 . Nice work again. 
Maybe we can add a different, where the stored procedure works (read or write) with cache data. Thank you for contribution.

Reply:
I don't think this test needs to be too complicated. The purpose of this issue is for OGM to be able to call stored procedures on Infinispan. Once you call a stored procedure and you can read the value it returns, we don't really care where the value is coming from. We might want to add an additional test in the integration test module to check for storedprocudures returning values previously inserted in the cache, but it can wait and we can make it in a separate issue.